r-package/grf/vignettes/llf.Rmd

```diff
@@ -0,0 +1,147 @@
+---
+title: "Introduction to local linear forests"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{llf}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+set.seed(123)
+```
+
+```{r setup, warning = FALSE, message = FALSE}
+library(grf)
+library(glmnet)
+library(ggplot2)
+```
+
+This document aims to show how to implement local linear forests (LLF). We begin with the standard use case, walking through parameter choices and method details, and then discuss how to use local linear corrections with larger datasets. 
```

Comment:
[nitpick] The tutorial will show you how to _use_ local linear forests. The implementation was done by you.